remote/src/main/java/com/playone/mobile/remote/PlayoneServiceFirebaseImpl.kt

```diff
@@ -11,7 +12,12 @@ import com.playone.mobile.remote.model.UserModel
 class PlayoneServiceFirebaseImpl(
     private val playoneFirebase: PlayoneFirebase
 ) : PlayoneService {
-    override fun fetchPlayoneList(userId: Int) = TODO()
+    override fun fetchPlayoneList(userId: Int) =
+        single<List<PlayoneModel>> { emitter ->
+            playoneFirebase.getPlayoneList(userId, emitter::onSuccess) { code, msg, detail ->
+                emitter.onError(Exception("error code: $code, msg: $msg, detail: $detail"))
```

Comment:
I think we can declare our own Exception and passing what we need as parameters rather than compose a message string.

Reply:
That's a good idea. I'm not familiar to declare the Exception. If you have a good way, please let me know.